src/expr/src/scalar/func.rs

```diff
@@ -3144,6 +3173,7 @@ pub enum VariadicFunc {
         field_names: Vec<ColumnName>,
     },
     ListSlice,
+    SplitString,
```

Comment:
A bit strange to call this `SplitString` when it's called `split_part` everywhere else?